app/helpers/icons_helper.rb

```diff
@@ -114,6 +114,10 @@ def chat_message_icon
     fa_icon('commenting-o')
   end
 
+  def comment_icon
+    fa_icon('comment')
```

Comment:
Some helper functions make no sense to me. `fa_icon()` in the template should be finde. The name for the icons will never be changed by font-awesoem